src/store/modules/job/actions.ts

```diff
@@ -458,19 +458,17 @@ const actions: ActionTree<JobState, RootState> = {
     try {
       resp = await JobService.updateJob(payload)
       if (resp.status === 200 && !hasError(resp) && resp.data.successMessage) {
-        let jobs = await dispatch('fetchJobs', {
+        const jobs = await dispatch('fetchJobs', {
           inputFields: {
             'systemJobEnumId': payload.systemJobEnumId,
             'systemJobEnumId_op': 'equals'
           }
         })
-        if(jobs.status === 200 && !hasError(jobs) && jobs.data?.docs.length) {
-          jobs = jobs.data?.docs;
-          const job = jobs.find((job: any) => job?.jobId === payload.jobId);
+        const job = jobs[1];
```

Comment:
Why we are accessing values using index? As there is no gurantee that the specific job will come on that index only.

Reply:
Right sir. Fixed it.